{"uuid":"07d9ac90-8115-495c-9ee3-27ab0e145628","historyId":"71d7784c1861a43f0f028bca56628b17","fullName":"tests.ErrorValidationTest.testInvalidLoginLandingPage","labels":[{"name":"package","value":"tests.ErrorValidationTest"},{"name":"testClass","value":"tests.ErrorValidationTest"},{"name":"testMethod","value":"testInvalidLoginLandingPage"},{"name":"parentSuite","value":"RegressionTest"},{"name":"suite","value":"ErrorValidationTest"},{"name":"subSuite","value":"tests.ErrorValidationTest"},{"name":"host","value":"Qcentris-396"},{"name":"thread","value":"26116@Qcentris-396.TestNG-tests-2(33)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"TC13 - Validates error message for invalid login attempts on the landing page.","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.AssertionError","trace":"java.lang.AssertionError\r\n\tat org.junit.Assert.fail(Assert.java:87)\r\n\tat org.junit.Assert.assertTrue(Assert.java:42)\r\n\tat org.junit.Assert.assertTrue(Assert.java:53)\r\n\tat tests.ErrorValidationTest.testInvalidLoginLandingPage(ErrorValidationTest.java:71)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.access$000(SuiteRunner.java:28)\r\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)\r\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\r\n"},"stage":"finished","steps":[{"name":"Logging test details for Allure.","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"error description","source":"d3a8c984-48f5-4371-a40c-060e527bb811-attachment.txt","type":"text/plain"}],"parameters":[],"start":1741215052039,"stop":1741215052042}],"attachments":[],"parameters":[{"name":"arg0","value":"data.LoginLandingPageData@2d32622c"},{"name":"Test Case ID","value":"TC13"},{"name":"Description","value":"Validates error message for invalid login attempts on the landing page."},{"name":"Username","value":"invalidUserName"},{"name":"Password","value":""}],"start":1741215052038,"stop":1741215054830}